pkg/debug/start_debug.go

```diff
@@ -48,7 +46,7 @@ func startDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alte
 	deployment := *originalDeployment
 
 	if alternateImageValue != "" {
-		log.Printf("setting debug image to %s\n", alternateImageValue)
+		logging.Info(fmt.Sprintf("setting debug image to %s\n", alternateImageValue))
```

Comment:
No more need for `Sprintf()` when logging
```suggestion
		logging.Info("setting debug image to %s\n", alternateImageValue)
```